lib/src/model/connection_state_update.dart

```diff
@@ -1,14 +1,11 @@
 import 'package:flutter_reactive_ble/src/model/generic_failure.dart';
-import 'package:functional_data/functional_data.dart';
 import 'package:meta/meta.dart';
 
-part 'connection_state_update.g.dart';
 //ignore_for_file: annotate_overrides
 
 ///Status update for a specific BLE device.
 @immutable
-@FunctionalData()
```

Comment:
Are you getting rid of functional_data in this p/r?

Reply:
not anymore it was a temporary workaround but we are now using latest functional data which contains the fix.